zingolib/src/wallet/send.rs

```diff
@@ -143,11 +144,22 @@ impl LightWallet {
         )
         .map_err(BuildTransactionError::UnifiedSpendKey)?;
 
-        // We don't support zip320 yet. Only one step.
-        if proposal.steps().len() != 1 {
-            return Err(BuildTransactionError::ExchangeAddressesNotSupported);
+        match proposal.steps().len() {
+            1 => {
+                self.create_transaction_helper(sapling_prover, unified_spend_key, proposal)
+                    .await
+            }
+            2 /*TODO: check that this is actually tex*/ => self.create_transaction_helper(sapling_prover, unified_spend_key, proposal).await,
+            _ => panic!("TODO: error handle"),
```

Comment:
can we change the BuildTransactionError::ExchangeAddressesNotSupported variant to something else that we put here?

Reply:
Ready for resolution.